TCBApp/TCBApp.WebApi/Extensions/HttpContextExtension.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Net;
+using System.Text.Json;
+
+namespace TCBApp.WebApi.Extensions;
+
+public static class HttpContextExtension
+{
+    public static async Task<T> ConvertByModel<T>(this HttpContent content,HttpWebRequest requestBody)
```

Comment:
Http content noli Nima bo'lsa ha o'chirib yuboring

Reply:
to'g'rilandi